conformance/lang/expressions/literals/floating_point_literal.balt

```diff
@@ -0,0 +1,6092 @@
+
+    a = 235 .55; //  @error invalid DecimalFloatingPointNumber due to unnecessary space
+
+    a = 25 .;  //  @error invalid DecimalFloatingPointNumber due to unnecessary space
+
+    a = . 99;  //  @error invalid DecimalFloatingPointNumber due to unnecessary space
+
+    a = . 99;  //  @error invalid DecimalFloatingPointNumber due to unnecessary space
+
+    a = 25. e174; //  @error invalid DecimalFloatingPointNumber due to unnecessary space
+
+    a = 25.E 174; //  @error invalid DecimalFloatingPointNumber due to unnecessary space
+
+    a = 25 .E174; //  @error invalid DecimalFloatingPointNumber due to unnecessary space
+
+    a = 25.B174; //  @error invalid DecimalFloatingPointNumber due to incorrect ExponentIndicator
+
+    a = 243.32B174; //  @error invalid DecimalFloatingPointNumber due to incorrect ExponentIndicator
+
+    a = 243.32e 174; //  @error invalid DecimalFloatingPointNumber due to unnecessary space
+
+    a = 243.32 e1742; // @error invalid DecimalFloatingPointNumber due to unnecessary space
+
+    a = 245.32 E 1742; // @error invalid DecimalFloatingPointNumber due to unnecessary spaces
+
+    a = .99eabc; // @error invalid DecimalFloatingPointNumber due to invalid characters after the ExponentIndicator in Exponent
+
+    a = 25.B+1742; //  @error invalid DecimalFloatingPointNumber due to incorrect ExponentIndicator
+
+    a = 25.BCF+1742; //  @error invalid DecimalFloatingPointNumber due to incorrect ExponentIndicator
+
+    a = 25.TYU+GFH; // @error invalid DecimalFloatingPointNumber due to invalid Exponent
+
+    a = 25.TYUGFH; // @error invalid DecimalFloatingPointNumber due to rhs of decimal point not being digits
+
+    a = TYUGFH.23; // @error invalid DecimalFloatingPointNumber due to lhs of decimal point not being a DecimalNumber
+
+    a = TYUGFH.; // @error invalid DecimalFloatingPointNumber due to lhs of decimal point not being a DecimalNumber
+    
+    a = .TYUGFH; // @error invalid DecimalFloatingPointNumber due to rhs of decimal point not being digits
+    
+    a = 25..23; // @error invalid DecimalFloatingPointNumber due to multiple decimal point symbols 
+
+    a = 25.E+ty; // @error invalid DecimalFloatingPointNumber due to incorrect Exponent
+
+    a = 25.Ety;  // @error invalid DecimalFloatingPointNumber due to incorrect Exponent
+
+    a = 25.BC2F+1742; // @error invalid DecimalFloatingPointNumber due to incorrect Exponent
+
+    a = 25.BC2F+17VF42; // @error invalid DecimalFloatingPointNumber due to incorrect Exponent
+
+    a = 25.BC2?F+17VF?42; // @error invalid DecimalFloatingPointNumber due to incorrect Exponent
+
+    a = 25.67p+20; // @error invalid DecimalFloatingPointNumber due to incorrect ExponentIndicator
+}
+
+Test-Case: error
+Description: Test invalid decimal floating point numbers.
+Labels: decimal, float
+
+function init() {
+    float|decimal a = .99+1742; // @error invalid DecimalFloatingPointNumber due to missing ExponentIndicator
```

Comment:
Let's update the binding patterns wherever applicable to avoid unused variables.
```suggestion
    float|decimal _ = .99+1742; // @error invalid DecimalFloatingPointNumber due to missing ExponentIndicator
```

Reply:
[808331d](https://github.com/ballerina-platform/ballerina-spec/pull/947/commits/808331da7e85b86e4b950f73bcddde9b67a754dc)